settings/template.ini

```diff
@@ -72,6 +72,13 @@ Methods[]=ftps
 Methods[]=webdav
 Methods[]=webdavs
 
+# Add support for text which is started from www. without protocol.
+# For example change text "www.ez.no" to link tags.
+LinkWwwUrlsWithoutProtocol=disabled
+
+HideProtocols[]
+#HideProtocols[]=http
```

Comment:
Now for a more positive comment :) We like that you give a specific example of how to use this setting.